### PR TITLE
Preapps 4190

### PR DESCRIPTION
**Pull request by shardool-gore**
Preapps 4190

### CODE REVIEW

File: licenserequirements.adoc [hunk 1]
```diff
@@ -31,6 +38,10 @@ that can be created. The archive feature must be installed.
 A {product-short} license is required in order to create accounts in
 the {product-edition-commercial} of {product-name}.
 
+[IMPORTANT]
+A {product-short} license is required in order to use the {modern} in
```
Comment (abyshukla) [inline licenserequirements.adoc]: A {product-short} license is required to use the {modern} in

File: licenserequirements.adoc [hunk 2]
```diff
@@ -31,6 +38,10 @@ that can be created. The archive feature must be installed.
...
+the {product-edition-commercial} of {product-name}.
```
Comment (abyshukla) [inline licenserequirements.adoc]: {product-name}'s {product-edition-commercial}.

File: upgrade.adoc [hunk 3]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+endif::z9[]
+
+//Before you upgrade instructions for Zimbra 9
+ifdef::z9[]
+== Before you upgrade
+
+Please review the following information to decide if {product-name} {product-version}
+is suitable for you.
+
+* Zimbra Touch Client, Zimbra Mobile Client, Zimbra HTML (Standard) Client have been removed from {product-name} {product-version}
```
Comment (abyshukla) [inline upgrade.adoc]: "have been removed" is passive voice. Please consider:

> Zimbra Touch Client, Zimbra Mobile Client, Zimbra HTML (Standard) Client are no longer a part of {product-name} {product-version}.

File: upgrade.adoc [hunk 4]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+* {product-name} {product-version} currently does not support Rolling Upgrades
```
Comment (abyshukla) [inline upgrade.adoc]: full stop at the end
rolling upgrades (lowercase)

File: upgrade.adoc [hunk 5]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+* {product-name} {product-version} introduces two versions of {product-short} Web Client, namely, Modern and Classic
+** A {product-short} license is required in order to use the Modern Web Client in
```
Comment (abyshukla) [inline upgrade.adoc]: "in order to" is too wordy. 'to' is a better replacement.

File: upgrade.adoc [hunk 6]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+the {product-edition-commercial} of {product-name}.
```
Comment (abyshukla) [inline upgrade.adoc]: > {product-name}'s {product-edition-commercial}.

is crisper instead of 

> the {product-edition-commercial} of {product-name}.

File: upgrade.adoc [hunk 7]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+//Before you upgrade instructions for Zimbra 9
+ifdef::z9[]
+== Before you upgrade
+
+Please review the following information to decide if {product-name} {product-version}
+is suitable for you.
+
+* Zimbra Touch Client, Zimbra Mobile Client, Zimbra HTML (Standard) Client have been removed from {product-name} {product-version}
+* {product-name} {product-version} currently does not support Rolling Upgrades
+* {product-name} {product-version} introduces two versions of {product-short} Web Client, namely, Modern and Classic
```
Comment (abyshukla) [inline upgrade.adoc]: Please consider using 'em dash' instead of 'namely'
full stop at the end

> * {product-name} {product-version} introduces two versions of {product-short} Web Client -- Modern and Classic.

File: upgrade.adoc [hunk 8]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+** A {product-short} license is required in order to use the Modern Web Client in
+the {product-edition-commercial} of {product-name}.
+** To know more about highlights of the Modern Web Client, please refer to the *What's New* document for {product-name} {product-version}
+** Classic Web Client offers the same functionalities as offered by the Advanced Web Client in  {product-name} 8.8.15 version
+** Modern Web Client does not currently support some features like Personas, Briefcase and Dumpster. Some of these features will be added during upcoming releases. To view the detailed comparison of features supported in Modern Web Client, please refer to *Web Client Comparison* document
```
Comment (abyshukla) [inline upgrade.adoc]: full stop at the end of the sentence. Suggestion
> Modern Web Client does not currently support features like Personas, Briefcase, and Dumpster. Some of these features are part of upcoming releases. To view the detailed comparison of features supported in Modern Web Client, please refer to *Web Client Comparison* document.

File: upgrade.adoc [hunk 9]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+** Existing customized themes, logo branding changes, and crontab changes will not work with Modern Web Client. Branding needs to be re-configured to work with the Modern Web Client. Themes are currently not supported for Modern Web Client.
```
Comment (abyshukla) [inline upgrade.adoc]: Future tense and passive voice is avoided in technical writing. Suggestion
> ** Existing customized themes, logo branding changes, and crontab changes are incompatible with, and hence do not reflect in, the Modern Web Client. Branding needs to be re-configured to work with the Modern Web Client. The web client also does not currently support themes.

File: upgrade.adoc [hunk 10]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+** Zimlets are supported on both the Web Clients. 
+*** Existing Zimlets will continue to work in the Classic Web Client, however they will not work on Modern Web Client. There is no way to migrate the zimlets from Classic to Modern Web Client or vice-versa.
```
Comment (abyshukla) [inline upgrade.adoc]: Future tense is avoided in technical writing. Suggestion
> *** Existing Zimlets work with Classic Web Client; however, they are incompatible with Modern Web Client. Currently, there is no way to migrate the zimlets from Classic to Modern Web Client or vice-versa.

File: upgrade.adoc [hunk 11]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+** Classic Web Client offers the same functionalities as offered by the Advanced Web Client in  {product-name} 8.8.15 version
+** Modern Web Client does not currently support some features like Personas, Briefcase and Dumpster. Some of these features will be added during upcoming releases. To view the detailed comparison of features supported in Modern Web Client, please refer to *Web Client Comparison* document
+** Existing customized themes, logo branding changes, and crontab changes will not work with Modern Web Client. Branding needs to be re-configured to work with the Modern Web Client. Themes are currently not supported for Modern Web Client.
+** Zimlets are supported on both the Web Clients. 
+*** Existing Zimlets will continue to work in the Classic Web Client, however they will not work on Modern Web Client. There is no way to migrate the zimlets from Classic to Modern Web Client or vice-versa. 
+*** Zimbra Connect zimlet, Zimbra Drive zimlet and Zimbra Docs zimlet, are available for both the web clients. Seperate versions of these services need to be installed for Classic Web Client and Modern Web Client
```
Comment (abyshukla) [inline upgrade.adoc]: passive voice. Suggestion
> *** Zimbra Connect zimlet, Zimbra Drive zimlet, and Zimbra Docs zimlet, are available for both the web clients. You can install separate versions of these services for Classic Web Client and Modern Web Client.

File: upgrade.adoc [hunk 12]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+
+The following tasks might need to be performed before you
```
Comment (abyshukla) [inline upgrade.adoc]: passive voice. Suggestion
> You might need to perform the following tasks before the

File: upgrade.adoc [hunk 13]
```diff
@@ -520,14 +557,26 @@ change your configuration.  Configuration settings stored in _LDAP_ and
...
 Zimlets are enabled after the upgrade. Zimlets that you customized
 and/or deployed are preserved during the upgrade but will be
 disabled. As upgrading of customized Zimlets cannot be tested before
 the release, {product-provider} recommends that you verify that your customized
 Zimlets work correctly before re-enabling them for your end-users
 after the upgrade.
 
+ifdef::z9[]
+[IMPORTANT]
+Zimlets are supported on both Classic and Modern Web Clients. However, Zimlets used for Classic Web Client are not compatible with Modern Web Client. There is no way to migrate the zimlets from Classic to Modern Web Client or vice-versa. Existing zimlets need to be re-developed in order for them to work on Modern Web Client.
```
Comment (abyshukla) [inline upgrade.adoc]: Repeated

File: upgrade.adoc [hunk 14]
```diff
@@ -520,14 +557,26 @@ change your configuration.  Configuration settings stored in _LDAP_ and
 _localconfig_ are preserved during upgrades. Any files installed by
 {product-name} might be deprecated and/or overwritten during
 upgrades, removing any customizations. This includes customized
-themes, logo branding changes, and crontab changes.  Only the core
+themes, logo branding changes, and crontab changes.  
+
+ifdef::z9[]
+[IMPORTANT]
+Branding needs to be re-configured to work with the Modern Web Client. Themes are currently not supported on Modern Web Client.
```
Comment (abyshukla) [inline upgrade.adoc]: Repeated.

File: upgrade.adoc [hunk 15]
```diff
@@ -29,6 +38,34 @@ and to determine if you wish to install them.
...
+** Modern Web Client does not currently support some features like Personas, Briefcase and Dumpster. Some of these features will be added during upcoming releases. To view the detailed comparison of features supported in Modern Web Client, please refer to *Web Client Comparison* document
+** Existing customized themes, logo branding changes, and crontab changes will not work with Modern Web Client. Branding needs to be re-configured to work with the Modern Web Client. Themes are currently not supported for Modern Web Client.
+** Zimlets are supported on both the Web Clients. 
+*** Existing Zimlets will continue to work in the Classic Web Client, however they will not work on Modern Web Client. There is no way to migrate the zimlets from Classic to Modern Web Client or vice-versa. 
+*** Zimbra Connect zimlet, Zimbra Drive zimlet and Zimbra Docs zimlet, are available for both the web clients. Seperate versions of these services need to be installed for Classic Web Client and Modern Web Client
+
+The following tasks might need to be performed before you
+upgrade. After you review the tasks in this section, go to Upgrade
```
Comment (abyshukla) [inline upgrade.adoc]: "please"
> please  go to Upgrade

File: licenserequirements.adoc [hunk 16]
```diff
@@ -14,6 +14,13 @@ the number of accounts created are shown.
 the native mail mobile feature enabled.
 * **Touch Client accounts limit**. The maximum number of accounts that
 can have the touch client mobile feature enabled.
+
+ifdef::z9[]
++
+[IMPORTANT]
+Mobile and Touch clients have been removed from {product-name} {product-version}.
```
Comment (abyshukla) [inline licenserequirements.adoc]: passive voice. Suggestion
> Mobile and Touch clients are no longer a part of {product-name} {product-version}.

**Review by abyshukla — CHANGES_REQUESTED** [review]

File: licenserequirements.adoc (hunk 16 above)
Comment (shardool-gore) [reply licenserequirements.adoc]: agree. changed

File: upgrade.adoc (hunk 14 above)
Comment (shardool-gore) [reply upgrade.adoc]: we need the repetition here.

File: upgrade.adoc (hunk 5 above)
Comment (mogsdad) [reply upgrade.adoc]: Change "in order to" -> "to" throughout the document.

File: systemrequirements.adoc [hunk 17]
```diff
@@ -48,24 +48,23 @@ additional space for mail storage. The other nodes require 100MB.
 * CentOS Linux 6 (64-bit), *patch level 4 or later is required*
 * Oracle Linux 7.2
 * Oracle Linux 6.6
-* Ubuntu 16.04 LTS Server Edition (64-bit), starting from Zimbra Collaboration 8.7.1 and above
-* Ubuntu 18.04 LTS Server Edition (64-bit), starting from Zimbra Collaboration 8.8.15 and above
+* Ubuntu 16.04 LTS Server Edition (64-bit)
+* Ubuntu 18.04 LTS Server Edition (64-bit), starting from Zimbra Collaboration 8.8.12 and above
```
Comment (abyshukla) [inline systemrequirements.adoc]: Ubuntu 18.04 was GA with 8.8.15. With 8.8.12, it was still beta.

**Review by abyshukla — APPROVED** [review]
